source/class/qx/tool/migration/BaseMigration.js

```diff
@@ -93,23 +93,46 @@ qx.Class.define("qx.tool.migration.BaseMigration",{
     },
 
     /**
-     * Marks a migration step as applied
+     * Marks one or more migration steps as applied
+     * @param {Number|String} param Optional. If number, number of migrations to mark
+     * as applied, defaults to 1; if String, message to be `info()`ed if verbose=true
      */
-    markAsApplied() {
-      this.setApplied(this.getApplied()+1);
+    markAsApplied(param) {
+      let numberOfMigrations = 1;
+      if (typeof param == "string") {
+        if (this.getRunner().getVerbose()) {
+          this.info(param);
+        }
+      } else if (typeof param == "number") {
+        numberOfMigrations = param;
+      } else if (typeof param != "undefined") {
+        throw new TypeError("Argument must be string or number");
+      }
+      this.setApplied(this.getApplied()+numberOfMigrations);
     },
 
     /**
-     * Marks a migration step as pending
+     * Marks one or more migration steps as pending
+     * @param {Number|String} param Optional. If number, number of migrations to mark
+     * as applied, defaults to 1; if String, message to be `announce()`ed
```

Comment:
copy/paste error? `as applied` should be `as pending` here I think

Reply:
Yes 🙄 . Humanity will one day perish because of a copy & paste error in the nuclear launch application.

Reply:
Fixed.